tests/ci/create-osbuild.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -euo pipefail
+
+# following steps as noted on https://hackmd.io/ZbUOkeIXTjaP7XVpTSsrNw?view
+
+# Get OS data.
+source /etc/os-release
+
+ID=${ID:-}
+ARCH=$(arch)
+OS_VARIANT=${OS_VARIANT:-}
+IMAGE_TYPE=${IMAGE_TYPE:-}
+UUID=${UUID:-local}
+BOOT_LOCATION="http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/"
+
+
+
```

Comment:
Thanks @roobrien ! Would you mind adding comments explaining each part similar to what is described in the hackmd file ?

Reply:
I am new to pushing shared code to github, so not sure how to link change to this comment
https://github.com/osbuild/automotive-ci/pull/141/commits/b1b65c0fcbaa5376d77cd584c9fe38203f6736ec

Reply:
It automatically detected the change since you pushed changes for the same lines I had reviewed